lib/tag_editor.dart

```diff
@@ -149,14 +146,14 @@ class _TagsEditorState extends State<TagEditor> {
   }
 
   void _onSubmitted(String string) {
-    widget.onSubmitted(string);
+    widget.onSubmitted!(string);
     if (widget.resetTextOnSubmitted) {
       _textFieldController.text = '';
     }
   }
 
   /// Shamelessly copied from [InputDecorator]
-  Color _getDefaultIconColor(ThemeData themeData) {
+  Color? _getDefaultIconColor(ThemeData themeData) {
```

Comment:
Maybe you don't need to return an optional here, since `Brightness` only has dart/light, it should be ok if you just remove the default case (I didn't know why I put it there in the first place too 😓 ). `_getIconColor()` can also return just `Color` and not `Color?`

Reply:
Still a bit confused on this one, but I think allowing it to return null should work. The highest level it gets called allows Color? so it would default to the global theme if it comes out null, which should be acceptable.

Your edit shows you adding the ?, but my local version also has that, so I'm not totally sure how it ended up needing it added. I think I will accept your edit and verify locally in case there is a conflict..

Reply:
Updated to return null if themeData.brightness is somehow null (should never be). Might be worth reviewing how _getDefaultIconColor() and _getIconColor() are used overall, but I think with this change it will work reliably.

Reply:
Resolved in https://github.com/panuavakul/material_tag_editor/pull/23/commits/579c0e5d5a1752e25d31039ebe1d0578eb367a22

Reply:
Hi sorry, I didn't see this comment before. Since Theme.of(context) does not return an optional, I think it is safe to just remove the default case and then just make it so that it does not return optional. I changed that in this PR (https://github.com/panuavakul/material_tag_editor/pull/25).

Thanks again for the PR.